update_service/index.adoc

```diff
@@ -0,0 +1,48 @@
+:context: update-service
+[id="update-service"]
+= Understanding the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
```

Comment:
You need an intro sentence that sets the context for the assembly.

Reply:
Currently this assembly is "everything you need to be an Update Service admin", and I include `modules/update-service-overview.adoc` to give folks context about what the Update Service is about.  I'm not sure what other context to provide, unless we [shard this out into several assemblies][1], in which case I guess we could explain the current assembly vs. the other Update Service assemblies?  I'm agnostic about sharding into assemblies.  If you have a particular set of assemblies you'd like to see, just tell me what they are, and I'll pivot to that structure.

[1]: https://github.com/openshift/openshift-docs/pull/26219#discussion_r508037525